recipe/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+# Adopted from https://github.com/microsoft/vcpkg/blob/master/ports/jbigkit/CMakeLists.txt
+cmake_minimum_required(VERSION 3.4)
+project(libjbig C)
+
+set(SOURCES
+    "libjbig/jbig.c"
+    "libjbig/jbig_ar.c"
+    "libjbig/jbig85.c")
+
+set(HEADERS
+    "libjbig/jbig.h"
+    "libjbig/jbig_ar.h"
+    "libjbig/jbig85.h"
+)
+
+add_library(libjbig ${SOURCES})
```

Comment:
Why add a `lib` prefix?

Reply:
@isuruf Ah, I didn't really think of it is a problem at the first glance when I shamelessly get the recipe from `vcpkg`.  I removed the prefix now.